sdcm/sct_events/setup.py

```diff
@@ -60,6 +60,9 @@ def start_events_device(log_dir: Optional[Union[str, Path]] = None,
                                 event_class=DatabaseLogEvent.DATABASE_ERROR,
                                 regex=r'.*workload prioritization - update_service_levels_from_distributed_data: an '
                                       r'error occurred while retrieving configuration').publish()
+    EventsSeverityChangerFilter(new_severity=Severity.WARNING,
```

Comment:
It is possible that I'am wrong here, but AFAIK we have special way to do this:
1. Add DatabaseLogEvent subtype at database.py:

```
DatabaseLogEvent.add_subevent_type("EMPTY_NESTED_EXCEPTION", severity=Severity.WARNING,
                                   regex=r"cql_server - exception while processing connection: "
                                         r"seastar::nested_exception \(seastar::nested_exception\)$")

```

2. Add this subtype to SYSTEM_ERROR_EVENTS at database.py:

```
SYSTEM_ERROR_EVENTS = (
    ...,
    DatabaseLogEvent.EMPTY_NESTED_EXCEPTION(),
    ...,
)
```

Look at EMPTY_NESTED_EXCEPTION as example

Reply:
The idea was not add new error or warning Event, and use DATABASE_ERROR event with special regexp

Reply:
Yes, but with current approach if you want to keep certain log records separate for all tests you creating subtype for it, as it is done for tens other log records

Reply:
@enaydanov , Eugene what do you think of that ?

Reply:
If i understood correctly, @roydahan means this approach: 
`I understand that, but if there is not a case where this error matters to us (e.g. when adding node getting this error is bad and severity should be raised to error), this "error" pattern could be just added to the global ignore list and that's it.`

Reply:
If I understand correctly the case, if we want to create the event that already exists but with another severity, we need to change event severity by using `EventsSeverityChangerFilter` like Alex performed here